### DISCUSSION TITLE
Planning a Route

### DISCUSSION THREAD

**Discussion by wavard** [devemux86/cruiser#51] [Q&A] [answered]
I am using Cruiser V3.0.10. Previously a Kurviger user.

When I plan a route by setting the start & end. However, when trying to add a via or shaping point I get a message Route not Found even though the route is displayed.

I get the same message If I set the beginning & end of the route & try to extend the route.

What might I be doing wrong?

Is there a help page?

**Comment by devemux86** [comment, marked as answer]
Hi @wavard!

> even though the route is displayed

This is the already existing route.
It only changes after a new route is successfully calculated.

> when trying to add a via or shaping point I get a message Route not Found
> I get the same message If I set the beginning & end of the route & try to extend the route

What router is selected in "Settings | Routing | Routing"?
What options are selected in "Settings | Routing | Options"?
Do you have a specific route example?

Each router may have a different sensitivity about where we can set waypoints
and may require waypoints near the road and not on restricted roads for the selected travel mode.

**Comment by wavard** [comment]
Thanks devemux86 I have changed my Routing to Valhalla and Search to Graphopper. 
Adding Via points & Shaping point & extending routes now work as they are supposed to. I will keep an eye on it.
